### DISCUSSION TITLE
Reading records from the Browsers by using PyWinAuto??

### DISCUSSION THREAD

**Discussion by Fayzan-Bhatti** [pywinauto/pywinauto#1355] [Q&A]
Hi Guys. 
Thank you in advance. I am facing problem that I want to read all the records of the web-based applications. In which records will be saved in the tables/ List of records/ or will be forms. 
Like I have attached a screenshots in which we have vendor records in a website. 
<img width="902" alt="Screenshot 2023-12-01 at 3 49 29 PM" src="https://github.com/pywinauto/pywinauto/assets/75361430/a407d7ff-cdfe-48e0-b695-1bc114f9f311">
Like in the given picture I want to fetch all the no of items, quantity, price, amount. 
Is it possible to read all the records from the web-based UI and save the all record in my local/ server by using only **PyWinAuto**?

**Condition:** we cannot use any other library to read and write the records.

**Comment by vasily-v-ryabov** [comment]
It should be possible for standalone browser like Chrome, Firefox or IE.
For embedded Web UI you may have problems with enabling "accessibility" mode for the Web UI engine (need to ask developers to enable it). More details and examples are posted here: https://github.com/pywinauto/pywinauto/issues/590

Also Chrome requires special command line option to enable accessibility mode, while Firefox and IE enable it by default. It is mentioned on project wiki.

**Comment by vasily-v-ryabov** [comment]
Example for Chrome and Google Drive: https://github.com/vasily-v-ryabov/ui-automation-course/blob/master/02_google_drive_Murashov/test_explorer_google_drive.py
